Sophos/sophos firewall/ingest/parser.yml

```diff
@@ -32,18 +32,19 @@ stages:
 
           # HTTP
           http.request.referrer: "{{kv_event.message.referer}}"
-          http.response.status_code: "{{kv_event.message.httpresponsecode}}"
+          http.response.status_code: "{{kv_event.message.httpresponsecode or kv_event.message.statuscode}}"
           http.request.method: "{{kv_event.message.method}}"
           url.original: "{{kv_event.message.url}}"
           url.query: "{{kv_event.message.query}}"
           user_agent.original: "{{kv_event.user_agent}}"
 
           # General Network
-          source.bytes: "{{kv_event.message.sent_bytes}}"
+          source.bytes: "{{kv_event.message.sent_bytes or kv_event.message.size}}"
           source.packets: "{{kv_event.message.sent_pkts}}"
           destination.bytes: "{{kv_event.message.recv_bytes}}"
           destination.packets: "{{kv_event.message.recv_pkts}}"
           destination.domain: "{{kv_event.message.domain or kv_event.message.sni}}"
+          host.name: "{{kv_event.message.server}}"
```

Comment:
I think that `destination.domain` is more suitable. `host.name` is reserved for the name of the device from which the event originates.